src/routes/transactions/mappers/multisig-transactions/multisig-transaction-note.mapper.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { multisigTransactionBuilder } from '@/domain/safe/entities/__tests__/multisig-transaction.builder';
+import { MultisigTransactionNoteMapper } from '@/routes/transactions/mappers/multisig-transactions/multisig-transaction-note.mapper';
+
+const mapper = new MultisigTransactionNoteMapper();
+
+describe('Multisig Transaction note mapper (Unit)', () => {
+  it('should parse transaction `origin` and return a note', () => {
+    const transaction = multisigTransactionBuilder()
+      .with('origin', '{"name":"{\\"note\\":\\"This is a note\\"}"}')
```

Comment:
Nit: While it might not add significant value here, for the sake of consistency, I would use `faker.js` to generate input data for my tests.